Maple2.Server.Game/PacketHandlers/MapleopolyHandler.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Maple2.Database.Storage;
+using Maple2.Model.Enum;
+using Maple2.Model.Error;
+using Maple2.Model.Game;
+using Maple2.Model.Game.Event;
+using Maple2.Model.Metadata;
+using Maple2.PacketLib.Tools;
+using Maple2.Server.Core.Constants;
+using Maple2.Server.Core.PacketHandlers;
+using Maple2.Server.Game.Packets;
+using Maple2.Server.Game.Session;
+
+namespace Maple2.Server.Game.PacketHandlers;
+
+public class MapleopolyHandler : PacketHandler<GameSession> {
+    public override RecvOp OpCode => RecvOp.Mapleopoly;
+
+    #region Autofac Autowired
+    // ReSharper disable MemberCanBePrivate.Global
+    public required ItemMetadataStorage ItemMetadata { private get; init; }
+    public required GameStorage GameStorage { private get; init; }
+    // ReSharper restore All
+    #endregion
+
+    private enum Command : byte {
+        Load = 0,
+        Roll = 1,
+        Result = 3,
+    }
+
+    public override void Handle(GameSession session, IByteReader packet) {
+        using GameStorage.Request db = GameStorage.Context();
```

Comment:
I would move this all into a helper function and call it in the respective Handle functions. This way we don't run this code for invalid `Command`.